### ISSUE TITLE
Another PAR Issue

### ISSUE THREAD

**Issue by PostPollux** [MrKepzie/Natron#1123]
When I disable a resize node, the Pixel Aspect Ratio is wrong for the following nodes.
https://goo.gl/photos/o84pmYtBoYAGB89u5

**Comment by PostPollux** [comment]
Not fixed. Still the same with 75cd21f. At least at my machine...

**Comment by devernay** [comment]
Reopening

**Comment by MrKepzie** [comment]
it's fixed since yesterday

**Comment by devernay** [comment]
@PostPollux is it fixed?

**Comment by PostPollux** [comment]
No it's not fixed I tested at 52c1e7d.

Steps to reproduce:
Create a reader. Add a resize and choose 2k Cinemascope as format. Then add a grade node.
Now connect the viewer to the grade node. Everything fine until now. You will see a 2k Cinemascope image.

Now DISABLE the resize node. The result should now be exactly the same as if you connected the viewer directly to the read node. But the the result is a stretched version in the right format loosing half of the picture....
Just like I showed in the video I posted in the first post: https://goo.gl/photos/o84pmYtBoYAGB89u5
